pom.xml

```diff
@@ -37,10 +44,8 @@
     <version.assertj>3.5.2</version.assertj>
     <version.jcip.annotations>1.0</version.jcip.annotations>
     <version.shrinkwrap.resolver>2.2.5</version.shrinkwrap.resolver>
+    <version.snakeyaml>1.17</version.snakeyaml>
```

Comment:
We don't use `snakeyaml` programmatically anywhere in the code - hence there is no need to pull it as dependency as it won't change anything.